spacewalk/setup/spacewalk-setup.spec

```diff
@@ -192,6 +195,10 @@ mkdir -p $RPM_BUILD_ROOT%{_mandir}/man8
 /usr/bin/pod2man --section=1 $RPM_BUILD_ROOT/%{_bindir}/spacewalk-setup-sudoers| gzip > $RPM_BUILD_ROOT%{_mandir}/man1/spacewalk-setup-sudoers.1.gz
 /usr/bin/pod2man --section=1 $RPM_BUILD_ROOT/%{_bindir}/spacewalk-setup-ipa-authentication| gzip > $RPM_BUILD_ROOT%{_mandir}/man1/spacewalk-setup-ipa-authentication.1.gz
 
+# create directories for standalone formulas
+mkdir -p %{buildroot}/usr/share/salt-formulas/metadata
```

Comment:
We also need to create the main `/usr/share/salt-formulas/` directory here.

I would suggest to align this with how we're doing in the Salt package. Something like this:

```
# Directories for packaged Salt formulas
install -Dd -m 0750 %{buildroot}%{_prefix}/share/salt-formulas
install -Dd -m 0750 %{buildroot}%{_prefix}/share/salt-formulas/states
install -Dd -m 0750 %{buildroot}%{_prefix}/share/salt-formulas/metadata
```

Btw, the `%{_prefix}` macro should be also available here and resolving to `/usr/` AFAIU.

Also, don't forget to add those new directories at the end of the `%files` section of the spec file. Something like this:

```
%dir               %attr(0750, root, root) %{_prefix}/share/salt-formulas/
%dir               %attr(0750, root, root) %{_prefix}/share/salt-formulas/states/
%dir               %attr(0750, root, root) %{_prefix}/share/salt-formulas/metadata/
```

Hth! :smile:

Reply:
Thank you @meaksh for the review and for helping. I have adjusted the code. Please have a look.